src/main/java/Barry.java

```diff
@@ -0,0 +1,322 @@
+        System.out.println("\t [Type help for a list of available commands]");
+        System.out.println("\t____________________________________________________________\n");
+    }
+
+    // Exits the program with a message
+    private static void exit() {
+        System.out.println("\t____________________________________________________________");
+        System.out.println("\t Bye, see you next time! :)");
+        System.out.println("\t____________________________________________________________\n");
+    }
+
+    // List all available commands
+    private static void help() {
+        System.out.println("\t____________________________________________________________");
+        System.out.println("\t Here's a list of available commands:");
+        System.out.println("\t   help                                           List all available commands");
+        System.out.println("\t   bye                                            Quit");
+        System.out.println("\t   list                                           List all tasks in your list");
+        System.out.println("\t   mark [x]                                       Mark task number x in your list");
+        System.out.println("\t   unmark [x]                                     Unmark task number x in your list");
+        System.out.println("\t   delete [x]                                     Delete task number x in your list");
+        System.out.println("\t   todo [task name]                               Add a ToDo to your list");
+        System.out.println("\t   deadline [task name] /by [deadline]            Add a Deadline to your list");
+        System.out.println("\t   event [task name] /from [start] /to [end]      Add an Event to your list");
+        System.out.println("\t____________________________________________________________\n");
+    }
+
+    // Task list mode, allows user to enter tasks into a list, and print all tasks in the list
+    // currently with "list", exits with "bye"
+    private static void taskListMode() {
+        Scanner scanner = new Scanner(System.in);
+        final Pattern markPattern = Pattern.compile("^mark\\s(\\d+)$");
+        final Pattern unmarkPattern = Pattern.compile("^unmark\\s(\\d+)$");
+        final Pattern deletePattern = Pattern.compile("^delete\\s(\\d+)$");
+        final Pattern todoPattern = Pattern.compile("^todo\\s(.+)$");
+        final Pattern deadlinePattern = Pattern.compile("^deadline\\s(.+)\\s/by\\s(.+)$");
+        final Pattern eventPattern = Pattern.compile("^event\\s(.+)\\s/from\\s(.+)\\s/to\\s(.+)$");
+        // Execution loop
+        while (true) {
+            // Command line input, stripped of whitespaces
+            String inputText = scanner.nextLine().strip();
+            if (inputText.equals("bye")) {
+                // Exit task list mode
+                break;
+            } else if (inputText.equals("help")) {
+                // List all available commands
+                help();
```

Comment:
LGTM. I like the way you doing here. Using the help() method from class to doing listed down all the commands. Consider the others commands to make your code shorter here.